### ISSUE TITLE
Boston.gov breadcrumb styling

### ISSUE THREAD

**Issue by reillyzlab** [CityOfBoston/design-team-workflow#1199]
Review current breadcrumbs on Drupal and advise whether they should be changed or not, specifically thinking about accessibility.

Following ticket dependent on that: https://github.com/CityOfBoston/boston.gov-d8/issues/354
